Files/ViewModels/ItemViewModel.cs

```diff
@@ -703,8 +700,7 @@ public bool IsLoadingItems
 
         public async void UnloadItemThumbnail(ListedItem item)
         {
-            await unloadIconSemaphore.WaitAsync();
```

Comment:
Is the semaphore necessary?

Reply:
My original intentions were to limit the item unloading so recycling a large amount of items wouldn't slow scrolling, but it looks like this isn't needed!